chp07_CA/Exercise_7_01_WolframCA_randomizedrules/CA.py

```diff
@@ -0,0 +1,92 @@
+# The Nature of Code
+# Daniel Shiffman
+# http://natureofcode.com
+
+# Wolfram Cellular Automata
+
+# A class to manage the CA
+
+class CA():
+
+    w = 5
```

Comment:
This is a constant and should be named WIDTH or whatever w actually means.

Reply:
This still needs to be addressed.

Reply:
Great! Will do it.